benchmarks/distributed/rpc/parameter_server/trainers/RpcTrainerBase.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch.distributed.rpc as rpc
+
+from .TrainerBase import TrainerBase
+
+
+class RpcTrainerBase(TrainerBase):
+
+    RPC_FUT_METRIC = "rpc_fut_metric"
+    RPC_FUT = "rpc_fut"
+    RPC_METRIC = "rpc_metric"
+    RPC = "rpc"
+
+    def __init__(self, rank):
+        super().__init__(rank)
+
+    def send_async_request(self, key, rref, server_method, *args, cuda=True):
```

Comment:
the name does not seem to indicate what this is doing. IIUC, the main purpose of this wrapper is to profile the delay?

Reply:
it sends the request and adds a metric for the future delay. it is a bit verbose for sending an async request. I will get rid of the method.